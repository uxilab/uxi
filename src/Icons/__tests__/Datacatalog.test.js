import React from 'react';
import Datacatalog from '../Datacatalog';

describe('Icon : <Datacatalog />', () => {
  it('match snapshot', () => {
    expect(shallow(<Datacatalog />)).toMatchSnapshot();
  });
});

