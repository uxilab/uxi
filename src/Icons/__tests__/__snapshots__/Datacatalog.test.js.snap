// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon : <Datacatalog /> match snapshot 1`] = `
<SvgIcon
  color="currentColor"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  size={24}
  style={Object {}}
  viewBox="0 0 24 24"
>
  <svg
    focusable="false"
    height="24px"
    viewBox="0 0 16.44 16"
    width="24px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="Layer 2"
      id="Layer_2"
    >
      <g
        data-name="Layer 1"
        id="Layer_1-2"
      >
        <path
          d="M9.22,10.09c1.49,0,3-.45,3-1.3V4.73c0-.85-1.5-1.3-3-1.3s-3,.45-3,1.3V8.79C6.23,9.64,7.73,10.09,9.22,10.09ZM9.22,4c1.5,0,2.42.48,2.42.73s-.92.73-2.42.73S6.8,5,6.8,4.73,7.72,4,9.22,4ZM6.8,5.52l.08.05A5.34,5.34,0,0,0,9.22,6a5.38,5.38,0,0,0,2.34-.46l.08-.05v.57c0,.25-.92.72-2.42.72S6.8,6.34,6.8,6.09Zm0,1.36.08,0a5.34,5.34,0,0,0,2.34.47,5.34,5.34,0,0,0,2.34-.47l.08,0v.56c0,.26-.92.73-2.42.73S6.8,7.7,6.8,7.44Zm0,1.35.08,0a5.34,5.34,0,0,0,2.34.47,5.34,5.34,0,0,0,2.34-.47l.08,0v.56c0,.26-.92.73-2.42.73S6.8,9.05,6.8,8.79Z"
        />
        <path
          d="M16.44.65A.65.65,0,0,0,15.79,0H4.3A2.3,2.3,0,0,0,2,2.3v.92H1.73a1.73,1.73,0,0,0-1.15,3A1.73,1.73,0,0,0,0,7.52,1.75,1.75,0,0,0,.58,8.81,1.71,1.71,0,0,0,0,10.09a1.74,1.74,0,0,0,1.73,1.73H2V13.7A2.3,2.3,0,0,0,4.3,16H15.79a.64.64,0,0,0,.4-.15h.1v-.1a.64.64,0,0,0,.15-.4ZM3.31,13.25a1,1,0,0,1,.92-.56h10.9v2H4.3a1,1,0,0,1-1-1Zm1-11.94H15.13V11.69H4.23a2,2,0,0,0-.92.22V2.3A1,1,0,0,1,4.3,1.31ZM2,8.34H1.73a.82.82,0,0,1-.81-.82.81.81,0,0,1,.81-.81H2Zm-.27-4.2H2V5.77H1.73A.81.81,0,0,1,.92,5,.82.82,0,0,1,1.73,4.14Zm0,6.76a.81.81,0,0,1-.81-.81.82.82,0,0,1,.81-.82H2V10.9Z"
        />
      </g>
    </g>
  </svg>
</SvgIcon>
`;
